apis/management.cattle.io/v3/k8s_windows_default.go

```diff
@@ -62,197 +61,160 @@ var (
 			KubernetesBinaries: m("rancher/hyperkube:v1.12.1-nanoserver-1803"),
 			FlannelCNIBinaries: m("rancher/flannel-cni:v0.0.1-nanoserver-1803"),
 			KubeletPause:       m("rancher/kubelet-pause:v0.0.1-nanoserver-1803"),
 		},
 		"v1.12.3-rancher1-1": {
-			NginxProxy:         m("rancher/nginx-proxy:v0.0.1-nanoserver-1803"),
 			KubernetesBinaries: m("rancher/hyperkube:v1.12.3-nanoserver-1803"),
 			FlannelCNIBinaries: m("rancher/flannel-cni:v0.0.1-nanoserver-1803"),
 			KubeletPause:       m("rancher/kubelet-pause:v0.0.1-nanoserver-1803"),
 		},
 		"v1.12.4-rancher1-1": {
-			NginxProxy:         m("rancher/nginx-proxy:v0.0.1-nanoserver-1803"),
 			KubernetesBinaries: m("rancher/hyperkube:v1.12.4-nanoserver-1803"),
 			FlannelCNIBinaries: m("rancher/flannel-cni:v0.0.1-nanoserver-1803"),
 			KubeletPause:       m("rancher/kubelet-pause:v0.0.1-nanoserver-1803"),
 		},
 		"v1.12.5-rancher1-2": {
-			NginxProxy:         m("rancher/nginx-proxy:v0.0.1-nanoserver-1803"),
 			KubernetesBinaries: m("rancher/hyperkube:v1.12.5-nanoserver-1803"),
 			FlannelCNIBinaries: m("rancher/flannel-cni:v0.0.1-nanoserver-1803"),
 			KubeletPause:       m("rancher/kubelet-pause:v0.0.1-nanoserver-1803"),
 		},
 		"v1.12.6-rancher1-2": {
-			NginxProxy:         m("rancher/nginx-proxy:v0.0.1-nanoserver-1803"),
 			KubernetesBinaries: m("rancher/hyperkube:v1.12.6-nanoserver-1803"),
-			FlannelCNIBinaries: m("rancher/flannel-cni:v0.0.1-nanoserver-1803"),
+			FlannelCNIBinaries: m("rancher/flannel-cni:v0.0.2-nanoserver-1803"),
 			KubeletPause:       m("rancher/kubelet-pause:v0.0.1-nanoserver-1803"),
 		},
 		"v1.13.1-rancher1-2": {
-			NginxProxy:         m("rancher/nginx-proxy:v0.0.1-nanoserver-1803"),
 			KubernetesBinaries: m("rancher/hyperkube:v1.13.1-nanoserver-1803"),
 			FlannelCNIBinaries: m("rancher/flannel-cni:v0.0.1-nanoserver-1803"),
 			KubeletPause:       m("rancher/kubelet-pause:v0.0.1-nanoserver-1803"),
 		},
 		"v1.13.4-rancher1-1": {
-			NginxProxy:         m("rancher/nginx-proxy:v0.0.1-nanoserver-1803"),
 			KubernetesBinaries: m("rancher/hyperkube:v1.13.4-nanoserver-1803"),
 			FlannelCNIBinaries: m("rancher/flannel-cni:v0.0.1-nanoserver-1803"),
-			CalicoCNIBinaries:  m("rancher/calico-cni:v0.0.1-nanoserver-1803"),
-			CanalCNIBinaries:   m("rancher/canal-cni:v0.0.1-nanoserver-1803"),
 			KubeletPause:       m("rancher/kubelet-pause:v0.0.1-nanoserver-1803"),
 		},
 		"v1.13.4-rancher1-2": {
-			NginxProxy:         m("rancher/nginx-proxy:v0.0.1-nanoserver-1803"),
 			KubernetesBinaries: m("rancher/hyperkube:v1.13.4-nanoserver-1803"),
-			FlannelCNIBinaries: m("rancher/flannel-cni:v0.0.1-nanoserver-1803"),
-			CalicoCNIBinaries:  m("rancher/calico-cni:v0.0.1-nanoserver-1803"),
-			CanalCNIBinaries:   m("rancher/canal-cni:v0.0.1-nanoserver-1803"),
+			FlannelCNIBinaries: m("rancher/flannel-cni:v0.0.2-nanoserver-1803"),
 			KubeletPause:       m("rancher/kubelet-pause:v0.0.1-nanoserver-1803"),
 		},
```

Comment:
https://github.com/rancher/types/blob/master/apis/management.cattle.io/v3/k8s_defaults.go#L27-L32

The current default version are `v1.11.8-rancher1-1`,  `v1.12.6-rancher1-2` and `v1.13.4-rancher1-2`.

And we only want to focus on v1.14 which is not available in Rancher yet. So I think you don't need to change the code for other versions. Let's just update it for `v1.13.4-rancher1-2`.

Reply:
Updated